tests/test_analyze_docker_dockerfile.py

```diff
@@ -121,8 +121,13 @@ def testFindGitInfo(self):
         sample_line = 'ADD plain_file /tmp'
         comment_line = dockerfile.find_git_info(sample_line, self.pin_add)
         project_name = comment_line.split(',')[0]
+        url_info = comment_line.split(',')[2].strip(' ')
         self.assertIn(project_name, ['git project name: project',
                                      'git project name: tern'])
+        self.assertIn(url_info,
+                      ['url info: origin:https://github.com/vmware/tern '
+                       'upstream:https://github.com/vmware/tern.git',
+                       'url info: (not found)'])
```

Comment:
I may have missed something obvious here but why would url be `(not found)`?

Reply:
In the circleci test, the project folder seems not to have the git project url. In github `Pull Request Lint and Test / PRs (pull_request)` test, it has the git project url. Since we need to pass both check, i use this form of test, like the test for git project name which is in the form of assertIn.

Reply:
OK, I got the CircleCI issue. I think there should be a better way to test this. Let me get back to you on this one.

Reply:
I tried running this test locally on my clone of tern and I get `url info: (not found)` which shouldn't happen. It should be something like `url info: https://github.com/nishakm/tern, https://github.com/vmware/tern`.

Reply:
I ran a test on running `git remote -v` on CircleCI and GHA. It looks like the difference is just the addition of the upstream project in the GHA environment, which is expected because all the tests are run in the same environment there.
CircleCI:
```
#!/bin/bash -eo pipefail
python ci/test_commit_message.py

origin	git@github.com:vmware/tern.git (fetch)
origin	git@github.com:vmware/tern.git (push)

Commit message checks pass
```
GHA:
```

Run python ci/test_commit_message.py
origin	https://github.com/vmware/tern (fetch)
origin	https://github.com/vmware/tern (push)
upstream	https://github.com/vmware/tern.git (fetch)
upstream	https://github.com/vmware/tern.git (push)

Commit message checks pass
```
Either way, you should get one url here: `https://github.com/vmware/tern`

Another issue is that if you are working on a fork of the main project, then `git remote -v` will return something like `https://github.com/<username>/tern.git`. In this case the assert will still have to pass. I wonder if you could instead check if the beginning string has `https://github.com` and the end says `tern`.

Reply:
Shoud we include ssh?like
`upstream	ssh://git@github.com/vmware/tern.git (fetch)`

Reply:
I don't think we should. In fact, I think not including the transport protocol (`http/https/git`) would be better. So `get_git_url` should produce `['github.com/vmware/tern']` for the CircleCI and GHA case and `['github.com/vmware/tern', 'github.com/<username>/tern']` for the local development case.